Gordon360/Services/RecIM/ActivityService.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.CCT.Context;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+
+namespace Gordon360.Services.RecIM
+{
+    public class ActivityService : IActivityService
+    {
+        private readonly CCTContext _context;
+
+        public ActivityService(CCTContext context)
+        {
+            _context = context;
+        }
+
+
+        public IEnumerable<Activity> GetActivities()
+        {
+            var leagues = _context.Activity
+                            .Include(l => l.Team)
+                            .Include(l => l.Series)
+                            .AsEnumerable();
```

Comment:
Currently these services are written with the views in mind so that a single service will call everything needed for everything needed on the page. 
For this instance, GetActivities would be used on the Activity History type page/ search where we will want to see each previous activity with all an expandable menu type component that would show all the teams participating in that activity with associated series (regular season/ playoff)

I'm not sure if this is the cleanest way to handle them, but it is what our current concept is.